includes/EmailNotification.php

```diff
@@ -357,29 +357,16 @@ private function composeCommonMailtext() {
 		$keys = [];
 		$postTransformKeys = [];
 
-		if ( $this->isNewPage() ) {
-			// watchlist link tracking
-			list ( $keys['$NEWPAGE'], $keys['$NEWPAGEHTML'] ) = wfMsgHTMLwithLanguageAndAlternative (
-				'enotif_lastvisited',
-				'enotif_lastvisited',
-				F::app()->wg->LanguageCode,
-				[],
-				$this->title->getFullUrl( 's=wldiff&diff=0&previousRevId=' . $this->previousRevId )
-			);
-			$keys['$OLDID']   = $this->previousRevId;
-			$keys['$CHANGEDORCREATED'] = wfMessage( 'changed' )->inContentLanguage()->plain();
+		if ( $action == '' ) {
```

Comment:
Would empty() work here? Just asking, I really have no preference on this, but in the code we're using it more often that the empty string comparison